### PR TITLE
Issue #27 Add info about the pull request job

### PR DESCRIPTION
**Pull request by budhrg**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -64,6 +64,7 @@ For more information about enabling, disabling, and applying add-ons, see the li
 == CI Setup
 
 `minishift-addons` uses link:https://ci.centos.org/[CentOS CI] as CI build server.
+For a new pull request, it will trigger the link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] and if the pull request contains any change related to add-on, it will run the integration tests related to the add-ons.
```
Comment (LalatenduMohanty) [inline README.adoc]: @budhrg Somehow the  sentence is confusing. The confusion is  around pull request build job and running integration tests. Here is my suggestion:
`For a new pull request, the link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] will be triggered. But the build job runs the integration tests only when the pull request contains any change related to add-on.` WDYT?

File: README.adoc [hunk 2]
```diff
@@ -64,6 +64,7 @@ For more information about enabling, disabling, and applying add-ons, see the li
 == CI Setup
 
 `minishift-addons` uses link:https://ci.centos.org/[CentOS CI] as CI build server.
+For a new pull request, the link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] will be triggered. However, the build job runs the integration tests only when the pull request contains any change related to add-on.
```
Comment (anjannath) [inline README.adoc]: Any changes related to the feature file of the addon, we should be clear about this, cause AFAIU if and old addon is changed and no changes done to the feature file the PR job won't run the integration tests?

File: README.adoc [hunk 3]
```diff
@@ -64,6 +64,7 @@ For more information about enabling, disabling, and applying add-ons, see the li
 == CI Setup
 
 `minishift-addons` uses link:https://ci.centos.org/[CentOS CI] as CI build server.
+For a new pull request the link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] will be triggered. However, the build job runs the integration tests only when the pull request contains any change related to add-on.
```
Comment (LalatenduMohanty) [inline README.adoc]: s/`For a new pull request the pull request build job will be triggered`/`A pull request build job will be triggered for every pull requests`/

File: README.adoc [hunk 4]
```diff
@@ -64,7 +64,9 @@ For more information about enabling, disabling, and applying add-ons, see the li
 == CI Setup
 
 `minishift-addons` uses link:https://ci.centos.org/[CentOS CI] as CI build server.
-link:https://ci.centos.org/job/minishift-addons-nightly-b2d/[Nightly job] runs daily at midnight.
+A link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] will be triggered for every pull request.
+However, the build job runs the integration tests only when the pull request contains any change related to add-on.
+Also, a link:https://ci.centos.org/job/minishift-addons-nightly-b2d/[nightly job] runs daily at midnight.
```
Comment (LalatenduMohanty) [inline README.adoc]: Nitpick s/Also, a /Also a/

**Pull request merged**